src/components/dialogs/filter/expert/expert-filter-utils.ts

```diff
@@ -58,41 +69,52 @@ export const getOperators = (fieldName: string, intl: IntlShape) => {
                 OPERATOR_OPTIONS.GREATER_OR_EQUALS,
                 OPERATOR_OPTIONS.LOWER,
                 OPERATOR_OPTIONS.LOWER_OR_EQUALS,
+                OPERATOR_OPTIONS.BETWEEN,
                 OPERATOR_OPTIONS.EXISTS,
             ].map((operator) => ({
                 name: operator.name,
                 label: intl.formatMessage({ id: operator.label }),
             }));
         case DataType.BOOLEAN:
         case DataType.ENUM:
-            return [OPERATOR_OPTIONS.EQUALS, OPERATOR_OPTIONS.NOT_EQUALS].map(
-                (operator) => ({
-                    name: operator.name,
-                    label: intl.formatMessage({ id: operator.label }),
-                })
-            );
+            return [
+                OPERATOR_OPTIONS.EQUALS,
+                OPERATOR_OPTIONS.NOT_EQUALS,
+                OPERATOR_OPTIONS.IN,
+            ].map((operator) => ({
+                name: operator.name,
+                label: intl.formatMessage({ id: operator.label }),
+            }));
     }
     return defaultOperators;
 };
 
-export function getExpertRules(query: RuleGroupType): RuleGroupTypeExport {
-    function transformRule(rule: RuleType): RuleTypeExport {
+export function exportExpertRules(
+    query: CustomRuleGroupType
+): RuleGroupTypeExport {
+    function transformRule(rule: CustomRuleType): RuleTypeExport {
+        const isValueAnArray = Array.isArray(rule.value);
         return {
             field: rule.field as FieldType,
-            operator: rule.operator as OperatorType,
+            operator: Object.values(OPERATOR_OPTIONS).find(
+                (operator) => operator.name === rule.operator
+            )?.customName as OperatorType,
             value:
-                rule.operator !== OperatorType.EXISTS ? rule.value : undefined,
+                rule.operator !== OperatorType.EXISTS && !isValueAnArray
```

Comment:
That's maybe nitpicking, but maybe this to help readability ?

```suggestion
                !isValueAnArray && rule.operator !== OperatorType.EXISTS
```

Reply:
Yes ok